doc/decisions/0001-context-in-translations.md

```diff
@@ -0,0 +1,84 @@
+# 1. Context in Translations
+
+## Context
+
+For some translatable strings we create a message context (`msgctxt`) that contains the question text. This enables strings to be translated in the context of the question being asked. There are two current issues with this approach:
+
+- For some elements the context is incorrect. For example, the context for an answer guidance title is "Answer for: {question_text}".
+- Most translatable strings at the question level and below don't have context. This causes problems when the English text doesn't vary between question variants but the translated text needs to - https://trello.com/c/JEdbgtlT/3538-e-translations-add-context-to-all-child-elements-of-a-question-m
+
+
+## Decision
+
+- Remove the current message context `Answer for: {question_text}`.
+- Use a new `Question: {question_text}` message context for all translatable properties within a question other than title. This allows any text within a question to vary based on the question. 
+- Add a new `Type: {property_type}` comment for every string extracted based on the values in the table below e.g. `Type: Question definition list item`. This helps a translator understand what they are being asked to translate. 
+
+  | JSON Path | Type | Notes |
+  |-----------|------|-------|
+  | `$.title` | Questionnaire title | |
+  | `$.legal_basis` | Questionnaire legal basis | |
+  | `$.messages` | Global answer error message | |
```

Comment:
```suggestion
  | `$.messages` | Global answer error messages | |
```

Reply:
The way we do extraction, each extracted string will be a message rather than a messages.

Reply:
Might be worthy of a note - messages implies plural